spec/support/authentication.rb

```diff
@@ -0,0 +1,11 @@
+module Sorcery
+  module TestHelpers
+    module Rails
+      module Integration
+        def login_user_post(user, password)
+          page.driver.post(user_sessions_url, { email: user.email, password: password}) 
```

Comment:
Line is too long. [88/80]<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.<br>Trailing whitespace detected.